### PR TITLE
Display more info about failed process with -v flag

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: examples/run.php [hunk 1]
```diff
@@ -53,6 +53,16 @@ function testFile(): int
     return exit_code('test -f unknown-file');
 }
 
+#[AsTask(description: 'Run a command that will fail')]
+function exception(): void
+{
+    if (!output()->isVerbose()) {
+        output()->writeln('Re-run with -v, -vv, -vvv to different output.');
```
Comment (pyrech) [inline examples/run.php:60]: for different output
or
to see different output

?

**Review by pyrech — APPROVED** [review]
Thanks (but tests are broken right now)

**Comment by lyrixx** [comment]
It should be okay.

Output Cleaning with a stack trace is a mess, but it works!

i have no idea 2

**Review by pyrech — APPROVED** [review]
LGTM, thanks

**Pull request merged**